rhv_checks.py

```diff
@@ -203,11 +203,32 @@ def check_datacenters_status(system, **kwargs):
         print("Ok: all datacenter(s) are in the OK state: {}".format(okay))
         sys.exit(0)
 
+
+def check_storage_domain_attached_status(system, **kwargs):
+    """ Check the usage of all the datastores on the host. """
+    okay, critical, all = [], [], []
+    storage_domains_service = system.api.system_service().storage_domains_service()
+    all_sd = storage_domains_service.list()
+    data_centers_service = system.api.system_service().data_centers_service()
+    all_dc = data_centers_service.list()
+
+    for dc in all_dc:
+        print(dc.id)
```

Comment:
Maybe add some text before printing the `id`. Just maybe say `Currently  processing datacenter: dc.name-dc.id`

Reply:
I would avoid using print until the check is completed, i.e. just before `sys.exit()` This will avoid the ids showing up on the service in the Shinken UI. It would be good to eventually implement a logger for this sort of stuff.

Reply:
@john-dupuy I will work on logger possibly, but right now I will keep print statement there, what's say .?